src/components/Profile/Ourmodal.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {
+  Button, Header, Modal, Form,
+} from 'semantic-ui-react';
+import ImageUpLoader from 'react-images-upload';
+import Avatar from './Avatar';
+import { AVATAR } from '../../constants';
+import { getToken } from '../../utils/auth';
+import { updateImage, saveProfile } from '../../redux/actions';
+
+class Edit extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: getToken().user.username,
```

Comment:
How are you getting the user from the `getToken` method?

Reply:
yes, once a user has logged in and his/her token stored. You can identify the user using the tokens stored...